volontulo/views.py

```diff
@@ -207,16 +203,11 @@ def offer_form(request, organization_id):
                 request.get_full_path(),
                 ''
             )
-            send_mail(
-                u'Zgłoszenie oferty na Volontulo',
-                u'ID oferty: {0}.'.format(offer.id),
-                ['administrators@volontuloapp.org'],
-                u'ID oferty: <a href="{0}{1}">{2}</a>.'.format(
-                    domain,
-                    reverse('show_offer', args=[offer.id]),
-                    offer.id
-                ),
-            )
+            send_mail('offer_creation', ['administrators@volontuloapp.org'], {
```

Comment:
IMO start of new lines should be in the same columns, EOL before 'offer_creation' and ['administrators@volontuloapp.org'].